### PR TITLE
feat(skeleton): add Skeleton component

### PR DESCRIPTION
**Pull request by 0x74616e67**
## Checklist

- [ ] Fix linting errors
- [ ] Tests have been added / updated (or snapshots)

### CODE REVIEW

File: components/skeleton/skeleton.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import { SkeletonAnimations, SkeletonVariants } from '../utils/prop-types'
+
+interface SkeletonBaseProps {
+  animation?: SkeletonAnimations | boolean
```
Comment (zzlune) [inline components/skeleton/skeleton.tsx]: maybe the variable 'none' is better than the boolean

File: pages/en-us/components/skeleton.mdx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+`}
+/>
+
+<Playground
+  title="Animation"
+  desc="Display active animation."
+  scope={{ Skeleton, Spacer }}
+  code={`
+  <>
+    <Skeleton animation={false} />
```
Comment (zzlune) [inline pages/en-us/components/skeleton.mdx]: same question: maybe the variable 'none' is better than the boolean I think

File: components/skeleton/skeleton.tsx (hunk 1 above)
Comment (AngeliaQ) [reply components/skeleton/skeleton.tsx]: agree

**Review by zzlune — DISMISSED** [review]

File: pages/zh-cn/components/skeleton.mdx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+<Attributes.Title>SkeletonVariants</Attributes.Title>
+
+```ts
+type SkeletonVariants = 'text' | 'rect' | 'circle'
+```
+
+<Attributes.Title>SkeletonAnimations</Attributes.Title>
+
+```ts
+type SkeletonAnimations = 'pulse' | 'wave' | 'false'
```
Comment (AngeliaQ) [inline pages/zh-cn/components/skeleton.mdx]: when none animation, it should be 'none'?

File: pages/en-us/components/skeleton.mdx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+<Attributes.Title>SkeletonVariants</Attributes.Title>
+
+```ts
+type SkeletonVariants = 'text' | 'rect' | 'circle'
+```
+
+<Attributes.Title>SkeletonAnimations</Attributes.Title>
+
+```ts
+type SkeletonAnimations = 'pulse' | 'wave' | 'false'
```
Comment (AngeliaQ) [inline pages/en-us/components/skeleton.mdx]: the same with below

**Review by AngeliaQ — CHANGES_REQUESTED** [review]

**Review by zzlune — APPROVED** [review]

**Review by AngeliaQ — APPROVED** [review]

**Pull request merged**